Uyanda.Coffee.Application/Features/BeverageManagement/Models/InvoiceModel.cs

```diff
@@ -12,6 +12,8 @@ public class InvoiceModel
 
         public IEnumerable<LineItemModel> LineItems { get; set; }
 
+        public IEnumerable<CurrencyModel> Currencies { get; set; }
```

Comment:
I don't think we will need a collection of objects listing prices of each item individually in another currency. It should sufice to know just the total cost and exchange rate.

Reply:
_Currency_ model and entity removed